magtogoek/adcp/process.py

```diff
@@ -651,6 +662,54 @@ def _load_navigation(dataset: xr.Dataset, navigation_files: str):
     return dataset
 
 
+def _regrid_dataset(dataset: xr.Dataset, pconfig: ProcessConfig) -> xr.Dataset:
```

Comment:
Notes: Uses attrs: ancillary_variables to fin QC vars.

Reply:
What should be added, removed or changed with respect to this comment?

Reply:
An intermediary variable was created from ancillary_variables for better readability.